src/stores/Tabs/tab.js

```diff
@@ -305,6 +309,14 @@ export const Tab = types
       self.semantic_search = semanticSearchList ?? [];
       return self.save();
     },
+    
+    setSemanticSearchThreshold(_min, _max) {
+      const min = _min ?? self.threshold.min;
+      const max = _max ?? self.threshold.max;
+
+      self.threshold = { min, max };
+      return self.save();
+    },
```

Comment:
I think it's better to not call this method with undefined parameters at all
```suggestion
    setSemanticSearchThreshold(min, max) {
      self.threshold = { min, max };
      return self.save();
    },
```